### PR TITLE
Bhavin.patel/remote config 1

### PR DESCRIPTION
**Pull request by bawsome**
### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/agent/guide/how_remote_config_works.md [hunk 1]
```diff
@@ -75,6 +75,11 @@ The following products and features are supported with Remote Config:
 
 - **Automatic default agent rule updates**: Automatically receive and update the default Agent rules maintained by Datadog as new Agent detections and enhancements are released. 
 
+### Observability Pipelines (OP)
+<div class="alert alert-info">This feature is in private beta.</div>
+
+- **Remotely update and deploy [Observability Pipelines Worker (OPW)][10]**: Build and update pipelines in the UI, remotely rolling out the new configuration to OPW running in your environment.
```
Comment (barieom) [inline content/en/agent/guide/how_remote_config_works.md:81]: Let's change to:

**Remotely deploy and update [Observability Pipelines Workers (OPW)][10]**: Build and edit pipelines in the Datadog UI, rolling out your configuration changes to OPW instances running in your environment.

**Review by barieom — COMMENTED** [review]
🚀

**Comment by bawsome** [comment]
Updated RC public doc page to include OP beta feature, updated RC wording throughout, and activation steps. 

The RC permission description will be added to RBAC permissions page through below PR: https://github.com/DataDog/dogweb/pull/93360

File: content/en/agent/guide/how_remote_config_works.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ Remote configuration is not available on the US1-FED Datadog site.
 <div class="alert alert-info">Remote Configuration is in beta.</a></div>
 
 ## Overview
-Remote Configuration is a Datadog capability that allows you to remotely configure the behavior of Datadog resources (for example, Agents and tracing libraries) deployed in your infrastructure, for select product features. Using Remote Configuration makes it easier to apply Agent and tracing library configurations in your environment on demand, decreasing management costs, reducing friction between teams, and accelerating resolution times. 
+Remote Configuration is a Datadog capability that allows you to remotely configure the behavior of Datadog resources (for example, Agents, tracing libraries, and Observability Pipelines Worker) deployed in your infrastructure, for select product features. Using Remote Configuration makes it easier to apply configurations to Datadog resources in your environment on demand, decreasing management costs, reducing friction between teams, and accelerating issue resolution times.
```
Comment (hestonhoffman) [inline content/en/agent/guide/how_remote_config_works.md:34]:
```suggestion
Remote Configuration is a Datadog capability that allows you to remotely configure the behavior of Datadog resources deployed in your infrastructure. You can configure select product features on your Datadog Agents, tracing libraries, and the Observability Pipelines Worker. Use Remote Configuration to apply configurations to Datadog resources in your environment on demand, decreasing management costs, reducing friction between teams, and accelerating issue resolution times.
```

File: content/en/agent/guide/how_remote_config_works.md [hunk 3]
```diff
@@ -147,11 +152,13 @@ After you perform these steps, your Agent requests its configuration from Datado
 [1]: /getting_started/site/
 [2]: /help/
 [3]: /account_management/api-app-keys/
-[4]: https://app.datadoghq.com/organization-settings/api-keys
+[4]: https://app.datadoghq.com/organization-settings/remote-config
 [5]: /security/default_rules/#cat-workload-security
 [6]: /tracing/trace_pipeline/ingestion_controls/#managing-ingestion-for-all-services-at-the-agent-level
 [7]: /dynamic_instrumentation/?tab=configurationyaml#enable-remote-configuration
 [8]: /security/application_security/how-appsec-works/#built-in-protection
+[9]: /account_management/rbac/permissions
```
Comment (JeanCoquelet) [inline content/en/agent/guide/how_remote_config_works.md:160]:
```suggestion
[9]: /account_management/rbac/permissions#access-management
```

File: content/en/agent/guide/how_remote_config_works.md [hunk 4]
```diff
@@ -147,11 +152,13 @@ After you perform these steps, your Agent requests its configuration from Datado
 [1]: /getting_started/site/
 [2]: /help/
 [3]: /account_management/api-app-keys/
```
Comment (JeanCoquelet) [inline content/en/agent/guide/how_remote_config_works.md:154]: NIT: I wonder if it should be 
```suggestion
[3]: /account_management/rbac/permissions#api-and-application-keys
```
as a link here, as we reference the api_key permission.

File: content/en/agent/guide/how_remote_config_works.md [hunk 5]
```diff
@@ -102,23 +107,23 @@ Datadog implements the following safeguards, designed to protect the confidentia
 | ASM 1-click activation        |               | 1.4.0         | 2.17.0        | 3.9.0         |
 
 ### Setup
-To enable Remote Configuration: 
+To activate Remote Configuration: 
 
-1.  [Contact support][2] to have your organization enabled for adding the Remote Configuration capability on API keys.
+1. Ensure your RBAC permissions include [`org_management`][9] so you can enable Remote Configuration for your organization.
```
Comment (hestonhoffman) [inline content/en/agent/guide/how_remote_config_works.md:112]:
```suggestion
1. Ensure your RBAC permissions include [`org_management`][9], so you can enable Remote Configuration for your organization.
```

**Review by JeanCoquelet — APPROVED** [review]
Left 2 comments, LGTM!

File: content/en/agent/guide/how_remote_config_works.md [hunk 6]
```diff
@@ -102,23 +107,23 @@ Datadog implements the following safeguards, designed to protect the confidentia
...
 
-2. Ensure your RBAC permissions include [`api_keys_write`][3] so you can create a new API key with the Remote Configuration scope, or add the scope on an existing API key. Contact your organization's Datadog administrator to update your permissions if you don't have it. You'll use the key with this scope to authenticate and authorize your Agent to use Remote Configuration. 
+2. Ensure your RBAC permissions include [`api_keys_write`][3] so you can create a new API key with the Remote Configuration capability, or add the capability on an existing API key. Contact your organization's Datadog administrator to update your permissions if you don't have it. You'll use the key with this capability to authenticate and authorize your Agent to use Remote Configuration.
```
Comment (hestonhoffman) [inline content/en/agent/guide/how_remote_config_works.md:114]:
```suggestion
2. Ensure your RBAC permissions include [`api_keys_write`][3], so you can create a new API key with the Remote Configuration capability, or add the capability to an existing API key. Contact your organization's Datadog administrator to update your permissions if you don't have it. A key with this capability allows you to authenticate and authorize your Agent to use Remote Configuration. 
```

File: content/en/agent/guide/how_remote_config_works.md [hunk 7]
```diff
@@ -102,23 +107,23 @@ Datadog implements the following safeguards, designed to protect the confidentia
...
 
-2. Ensure your RBAC permissions include [`api_keys_write`][3] so you can create a new API key with the Remote Configuration scope, or add the scope on an existing API key. Contact your organization's Datadog administrator to update your permissions if you don't have it. You'll use the key with this scope to authenticate and authorize your Agent to use Remote Configuration. 
+2. Ensure your RBAC permissions include [`api_keys_write`][3] so you can create a new API key with the Remote Configuration capability, or add the capability on an existing API key. Contact your organization's Datadog administrator to update your permissions if you don't have it. You'll use the key with this capability to authenticate and authorize your Agent to use Remote Configuration. 
 
-3. On the [Organization Settings][4] page, either select an existing API key, or create a new API key.
+3. On the [Remote Configuration][4] page, enable Remote Configuration. This enables Datadog resources across your organization to receive configurations from Datadog. 
 
-4. Enable Remote Config scope on the key:
+4. Select an existing API key or create a new API key, and enable Remote Config capability on the key:
```
Comment (hestonhoffman) [inline content/en/agent/guide/how_remote_config_works.md:118]:
```suggestion
4. Select an existing API key or create a new API key, and enable the Remote Config capability on the key:
```

**Review by arbll — APPROVED** [review]

File: content/en/agent/guide/how_remote_config_works.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (bawsome) [reply content/en/agent/guide/how_remote_config_works.md:34]: Yes, barring any better suggestions and how industry vendors classifies their "elements/resources/entities", I would say let's use resources.

Comment (bawsome) [reply content/en/agent/guide/how_remote_config_works.md:34]: Thx Heston for the review. On this one, the select features are not configured on Datadog Agents, tracing libraries, and OPW but rather the products themselves such as APM, ASM, etc. Also it's important to clarify what resources Datadog resources are inline. How about this?

Remote Configuration is a Datadog capability that allows you to remotely configure the behavior of Datadog resources (for example, Agents, tracing libraries, and Observability Pipelines Worker) deployed in your infrastructure, for select product features. Use Remote Configuration to apply configurations to Datadog resources in your environment on demand, decreasing management costs, reducing friction between teams, and accelerating issue resolution times.

**Review by hestonhoffman — APPROVED** [review]
LGTM. @bawsome is this ready to merge?

**Comment by bawsome** [comment]
> 

Hi Heston - @JeanCoquelet will give you the green signal (expected Monday) as that's when the associated UI changes will go live.

**Comment by JeanCoquelet** [comment]
@hestonhoffman this PR can be merged today 👍

**Pull request merged**